### PR TITLE
[SCR-3679] add documentation for begin-of and end-of date functions

### PR DESCRIPTION
**Pull request by Keraito**
initial documentation proposal for begin-of and end-of date functions. Please see the appropriate JIRA story cc @MariaShaposhnikova

### CODE REVIEW

**Comment by MariaShaposhnikova** [comment]
@Keraito , I've reviewed new function documents. Please double-check that everything is technically correct.

**Review by MariaShaposhnikova — APPROVED** [review]

File: content/en/docs/refguide/modeling/application-logic/expressions/begin-of-date-function-calls.md [hunk 1]
```diff
@@ -52,6 +52,12 @@ The output is:
 
 The `beginOfWeek` function calculates the beginning of the week compared to the initial date.
 
+{{% alert type="info" %}}
+
+The week begins on Sunday and ends on Saturday.
```
Comment (Keraito) [inline content/en/docs/refguide/modeling/application-logic/expressions/begin-of-date-function-calls.md]: @MariaShaposhnikova I made a mistake and had to change the implementation, it should now be something along the lines of: The beginning and the end of the week are based on the user's locale.

File: content/en/docs/refguide/modeling/application-logic/expressions/end-of-date-function-calls.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+```
+
+
+## 3 endOfWeek
+
+The `endOfWeek` function calculates the end of the week compared to the initial date.
+
+{{% alert type="info" %}}
+
+The week begins on Sunday and ends on Saturday.
```
Comment (Keraito) [inline content/en/docs/refguide/modeling/application-logic/expressions/end-of-date-function-calls.md]: Same as https://github.com/mendix/docs/pull/4819#discussion_r943270340

File: content/en/docs/refguide/modeling/application-logic/expressions/begin-of-date-function-calls.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+
+The output is:
+
+```java {linenos=false}
+"Wed Feb 07 00:00 CET 2007"
+```
+
+## 3 beginOfWeek
+
+The `beginOfWeek` function calculates the beginning of the week compared to the initial date. The beginning and the end of the week are based on the user's locale.
```
Comment (Keraito) [inline content/en/docs/refguide/modeling/application-logic/expressions/begin-of-date-function-calls.md]: One small additional piece of information: "The beginning and the end of the week are based on the user's locale. In the case of an anonymous user, the browser's locale is used instead.". Tried to voice this is an active way, but don't really know how to make it intuitive so I'll leave it to you.

File: content/en/docs/refguide/modeling/application-logic/expressions/end-of-date-function-calls.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+The output is:
+
+```java {linenos=false}
+"Wed Feb 07 23:59 CET 2007"
+```
+
+
+## 3 endOfWeek
+
+The `endOfWeek` function calculates the end of the week compared to the initial date. The beginning and the end of the week are based on the user's locale.
```
Comment (Keraito) [inline content/en/docs/refguide/modeling/application-logic/expressions/end-of-date-function-calls.md]: Same thing here: "The beginning and the end of the week are based on the user's locale. In the case of an anonymous user, the browser's locale is used instead."

**Pull request merged**